Makefile

```diff
@@ -335,26 +335,58 @@ shadow-server: export TARGET := clojure
 shadow-server:##@ Start shadow-cljs in server mode for watching
 	yarn shadow-cljs server
 
-test-watch: export TARGET := clojure
-test-watch: ##@ Watch tests and re-run no changes to cljs files
-	yarn install
-	nodemon --exec 'yarn shadow-cljs compile mocks && yarn shadow-cljs compile test && node --require ./test-resources/override.js target/test/test.js' -e cljs
+_clojure-test: export TARGET := clojure
+_clojure-test:
+	yarn install && \
+	yarn shadow-cljs compile mocks && \
+	yarn shadow-cljs compile test && \
+	node --require ./test-resources/override.js "$$SHADOW_OUTPUT_TO"
 
-test-watch-for-repl: export TARGET := clojure
-test-watch-for-repl: ##@ Watch tests and support REPL connections
-	yarn install
-	rm -f target/test/test.js
-	concurrently --kill-others --prefix-colors 'auto' --names 'build,repl' \
-		'yarn shadow-cljs compile mocks && yarn shadow-cljs watch test --verbose' \
-		'until [ -f ./target/test/test.js ] ; do sleep 1 ; done ; node --require ./test-resources/override.js ./target/test/test.js --repl'
+_clojure-test-watch: export TARGET := clojure
+_clojure-test-watch:
+	yarn install && \
+	yarn shadow-cljs compile mocks && \
+	nodemon --exec "yarn shadow-cljs compile test && node --require ./test-resources/override.js $$SHADOW_OUTPUT_TO" -e cljs
+
+test: export SHADOW_OUTPUT_TO := target/test/test.js
+test: export SHADOW_NS_REGEXP := .*-test$$
+test: ##@test Run all Clojure tests
+test: _clojure-test
```

Comment:
Shouldn't this be called `test-all` then for clarity?

Reply:
I wanted to keep the code more backwards compatible, so other devs not aware of these changes can keep their workflows.

Reply:
Fair enough.